apps/mobile/src/screens/settings/settings.tsx

```diff
@@ -45,28 +83,92 @@ const Settings: React.FC = () => {
 		},
 	];
 
+	const handleSaveAnswers = useCallback(
+		(payload: NotificationAnswersPayloadDto): void => {
+			void dispatch(userActions.saveNotificationAnswers(payload));
+		},
+		[dispatch],
+	);
+
+	const handleFormSubmit = useCallback((): void => {
+		void handleSubmit(handleSaveAnswers)();
+	}, [handleSubmit, handleSaveAnswers]);
+
 	return (
 		<ScreenWrapper edges={["top"]}>
-			<View
-				style={[
-					globalStyles.flex1,
-					globalStyles.justifyContentSpaceBetween,
-					globalStyles.p16,
-				]}
-			>
-				<Text>Settings!</Text>
-
-				<Button
-					iconLeftName="exit-to-app"
-					label="SIGN OUT"
-					onPress={handleModalShow}
+			<ScrollView contentContainerStyle={globalStyles.flex1}>
+				<View style={[globalStyles.flex1, globalStyles.p16, styles.container]}>
+					<Text preset="heading" style={globalStyles.mb24} weight="bold">
+						Settings
+					</Text>
+					<Text preset="subheading" style={globalStyles.mb12} weight="bold">
+						Days to receive tasks
+					</Text>
+
+					<View
+						style={[
+							globalStyles.flexDirectionRow,
+							globalStyles.justifyContentSpaceBetween,
+							globalStyles.gap8,
+							styles.checkboxWrapper,
+						]}
+					>
+						<MultipleCheckboxInput
+							checkAllLabel=""
+							checkboxStyle={[styles.checkbox, globalStyles.pv8]}
+							containerStyle={styles.checkboxContainer}
+							control={control}
+							errors={errors}
+							name="userTaskDays"
+							options={TASK_DAYS_OPTIONS}
+						/>
+					</View>
+
+					<Text
+						preset="subheading"
+						style={[globalStyles.mb12, globalStyles.mt24]}
+						weight="bold"
+					>
+						Notification frequency
+					</Text>
+
+					<View
+						style={[
+							globalStyles.flex1,
+							globalStyles.justifyContentSpaceBetween,
+						]}
+					>
+						<View style={[styles.radioContainer, globalStyles.pb32]}>
+							<RadioGroup
+								control={control}
+								errors={errors}
+								itemContainerStyle={globalStyles.mb16}
+								name="notificationFrequency"
+								options={NOTIFICATION_FREQUENCY_OPTIONS}
+							/>
+							<View style={globalStyles.mt32}>
+								<Button
+									appearance="outlined"
+									isDisabled={!isValid || !isDirty}
```

Comment:
To prevent multiple requests
```suggestion
									isDisabled={!isValid || !isDirty || isLoading}
```